### DISCUSSION TITLE
데모 준비

### DISCUSSION THREAD

**Discussion by kworkbee** [damilog/board-sample#92] [Show and tell]
_어떻게 더 많은 걸_ **잘** 보여줄 수 있을까?!

**Comment by MinChul-Son** [comment]
떨려욧!🤮

**Comment by kworkbee** [comment]
## 데모 시연 방법

> FE는 로컬에서 가동

[BE](https://api.g1tommy.me/matzip.html)
[MLServer](http://ml.g1tommy.me:8000/redoc)

**Comment by jihyunhillpark** [comment]
## 기능 목록
### ML
- 클러스터링 모델을 활용한 맛집 추천

### BE, FE
- 전체 맛집 게시판 조회 
  - 평점순 정렬
  - 카테고리별 조회
  - 페이징
- 단일 맛집 상세 게시물 조회
  - 조회수
  - 네이버 지도 링크
  - 네이버 평점
  - 댓글 평균 평점 계산 
- 단일 맛집 상세 게시물 작성
- 댓글 
  - 댓글 쓰기
  - 댓글 조회
  - 네이버 댓글 크롤링
  - 댓글 삭제
- 맛집 삭제

**Comment by kworkbee** [comment]
- 개요
  - 맛집 + 추천
- **추천 구현 방법**
  - 모델 선정 (특징 N개 / Kmodes clustering 통한 유사도 측정)
  - 데이터 수집 및 전처리 (크롤링)
- **서비스 구성 소개**
  - EC2 / ELB (Compute) + Route53 (DNS) + RDS (Database)
- 프론트엔드 / 백엔드 데모 시연
